site/content/docs/cli/command-line-reference.md

```diff
@@ -179,51 +179,52 @@ checkly env pull [FILENAME] [-f]
 ```bash
 npx checkly env pull -f
 ```
-Pull all environment variables to the `.env` file and overwrite it if it already exists.
+Pull all global variables to the `.env` file and overwrite it if it already exists.
 
 ### `npx checkly env ls`
 
-List global environment variables. This command does not list environment variables on group or check level.
+List global variables. This command does not list  variables on group or check level. For secrets, only the key name will be shown, not its value.
 
 ```bash
 checkly env ls
 ```
 
 ### `npx checkly env add`
 
-Add a global environment variable.
+Add a global variable.
 
 ```bash
-checkly env add [KEY] [VALUE] [-l]
+checkly env add [KEY] [VALUE] [-l | -s] 
 ```
 
 - `KEY`: Environment variable key.
 - `VALUE`: Environment variable value.
 - `--locked` or `-l`: Indicate that the environment variable will be locked, making it private to all read only users.
+- `--secret` or `-s`: Indicate that the environment variable will be created as a secret. 
 
 
 ### `npx checkly env update`
 
-Update a global environment variable.
+Update a global variable.
 
 ```bash
 checkly env update [KEY] [VALUE] [-l]
 ```
 
-- `KEY`: Environment variable key.
-- `VALUE`: Environment variable value.
-- `--locked` or `-l`: Indicate that the environment variable will be locked,  making it private to all read only users.
+- `KEY`: Variable key.
+- `VALUE`: Variable value.
+- `--locked` or `-l`: Indicate that the variable will be locked,  making it private to all read only users.
```

Comment:
During update teh `--secret` or `-s` is required, otherwise  it will fail

Reply:
Updated, have a look.